src/components/Comment.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { CommentType } from '../types/comment.type';
+
+const Comment = ({ comment }) => {
+    const [voteCounter, setVoteCounter] = useState(0);
+    const [isEmailVisible, setIsEmailVisible] = useState(false);
+
+    const handleClick = event => {
+        const shouldIncrease = event.target.value === 'increase';
+        const newCounterValue = shouldIncrease
+            ? voteCounter + 1
+            : voteCounter - 1;
+        setVoteCounter(newCounterValue);
+    };
+
+    const handleReveal = () => setIsEmailVisible(true);
+    const handleHide = () => setIsEmailVisible(false);
+
+    // Display details if:
+    // - identity consent was not supported when user posted,
+    // - identity consent was supported and user consented
+    //   to publish their details.
+    const displayDetails =
+        !('identityConsent' in comment) || comment.identityConsent;
+
+    return (
+        <li data-testid="comment">
+            <div>
+                <strong
+                    onMouseOver={() => handleReveal()}
+                    onFocus={() => handleReveal()}
+                    onMouseOut={() => handleHide()}
+                    onBlur={() => handleHide()}
```

Comment:
although `focus` and `blur` are extensively (and some people would say exclusively) used for input and buttons, I found it interesting that there's a use for other elements: https://stackoverflow.com/questions/3656467/is-it-possible-to-focus-on-a-div-using-javascript-focus-function/46115264#answer-3656524

Reply:
I wouldn't normally use it here, but eslint complained that if something has `onMouseOver` and/or `onMouseOver` then it also needs `onFocus` and `onBlur` so we cover also primarily keyboard users.

Reply:
I wouldn't normally use it here, but eslint complained that if something has `onMouseOver` and/or `onMouseOver` then it also needs `onFocus` and `onBlur` so we cover also primarily keyboard users.